### DISCUSSION TITLE
Does fluentd's syslog support Journal Namespaces?

### DISCUSSION THREAD

**Discussion by ogis-yamazaki** [fluent/fluentd#4181] [Q&A]
I'm using the systemd-journald namespace.

[Journal Namespaces]
https://www.freedesktop.org/software/systemd/man/systemd-journald.service.html#Journal%20Namespaces

I have prepared a namespace with the name ``broker``.

So I have two journals in /var/log/journal as follows

```
$ pwd
/var/log/journal/ec27f1ab87d25a7c63d1c9af177d59d7
$ ls -l
total 20
drwxr-sr-x+ 2 root systemd-journal 16384 May 18 16:42 ec27f1ab87d25a7c63d1c9af177d59d7
drwxr-sr-x+ 2 root systemd-journal  4096 May 18 16:35 ec27f1ab87d25a7c63d1c9af177d59d7.broker
```

Does the syslog plugin support more than 2 journals?

**Comment by daipom** [comment]
I'm not familiar with the systemd-journald namespace.

If you can send syslog to`in_syslog` somehow, it can receive it.

**Reply by ogis-yamazaki** [reply]
Thanks for your comment.

I was looking into in_syslog and realized that the plugin I am using is fluent-plugin-systemd.

I will submit a question to the fluent-plugin-systemd github and close the case.

**Comment by ogis-yamazaki** [comment]
https://github.com/fluent-plugin-systemd/fluent-plugin-systemd

This is the plugin I'm using.
Sorry for the way I asked the question.

**Reply by daipom** [reply]
No problem!
